src/Core/Mesh/TriangleMesh.inl

```diff
@@ -67,13 +67,13 @@ inline bool TriangleMesh::append( const TriangleMesh& other ) {
     // Deal with all attributes the same way (vertices and normals too)
     other.m_vertexAttribs.for_each_attrib( [this]( const auto& attr ) {
         if ( attr->isFloat() )
-            append_attrib<float>( attr );
+            this->append_attrib<float>( attr );

```

Comment:
why use this ptr ?

Reply:
because we are in the context of a labmda, and thus we can't just call the method without prepending by the object to call it on, which leads to a compilation error.